natives.json

```diff
@@ -111574,52 +111574,52 @@
 			"build": "1207"
 		},
 		"0x5E3BDDBCB83F3D84": {
-			"name": "_0x5E3BDDBCB83F3D84",
-			"comment": "",
+			"name": "_GIVE_WEAPON_TO_PED",
+			"comment": "Gives the ped the weapon. p7 is 0.5f, and p8 is 1.0f. condition is a scale of 0.0f to 1.0f. p11 and p12 are both 0. For a list of weapon groups, here is a weapons enum by Mooshe with all weapon-related hashes: https://pastebin.com/n72eW6zd",
 			"params": [
 				{
-					"type": "Any",
-					"name": "p0"
+					"type": "Ped",
+					"name": "ped"
 				},
 				{
-					"type": "Any",
-					"name": "p1"
+					"type": "uint",
+					"name": "weaponHash"
 				},
 				{
-					"type": "Any",
-					"name": "p2"
+					"type": "int",
+					"name": "ammoCount"
 				},
 				{
-					"type": "Any",
-					"name": "p3"
+					"type": "BOOL",
+					"name": "equipNow"
 				},
 				{
-					"type": "Any",
+					"type": "BOOL",
 					"name": "p4"
 				},
 				{
-					"type": "Any",
-					"name": "p5"
+					"type": "uint",
+					"name": "weaponGroup"
 				},
 				{
-					"type": "Any",
+					"type": "BOOL",
 					"name": "p6"
 				},
 				{
-					"type": "Any",
+					"type": "float",
 					"name": "p7"
 				},
 				{
-					"type": "Any",
+					"type": "float",
 					"name": "p8"
 				},
 				{
-					"type": "Any",
-					"name": "p9"
+					"type": "BOOL",
+					"name": "leftHanded"
 				},
 				{
-					"type": "Any",
-					"name": "p10"
+					"type": "float",
+					"name": "condition"
 				},
 				{
 					"type": "Any",
```

Comment:
And this is a ``BOOL``.

Reply:
hm, it actually seems this native is different from the PS4 decompiled scripts. In the PS4 version, there's only 12 parameters, and this parameter in particular is a float. I'll change it to an unknown bool for now, and just leave documentation on what I know works.